src/core/AppRouter/AdminSubRouter.tsx

```diff
@@ -65,6 +67,12 @@ export const AdminSubRouter: React.FC = () => (
       </Provided>
     </Route>
 
+    <Route path={[ADMIN_IA_WORLD_SCHEDULE]}>
```

Comment:
No need for array, it is useful only if multiple paths trigger the same route.
```suggestion
    <Route path={ADMIN_IA_WORLD_SCHEDULE}>
```